lib/index.coffee

```diff
@@ -93,7 +93,11 @@ module.exports = (opts) ->
     ###
 
     fetch_content = (type) ->
-      W client.entries(_.merge(type.filters, content_type: type.id))
+      W(
+        client.entries(
+          _.merge(type.filters, content_type: type.id, include: 10)
```

Comment:
Just seeing that nested linked objects weren't being resolved past level 2, and then looking around the docs for anything related to that. Really unsatisfying bug, because it was probably just an API change they launched without notifying anyone.